test/index.js

```diff
@@ -57,12 +55,7 @@ function compare (t, name, log) {
     })
 }
 
-function error (name, cb) {
+function error (name) {
   const html = fs.readFileSync(path.join(fixtures, `${name}.html`), 'utf8')
-
-  try {
-    return reshape({ parser }).process(html)
-  } catch (err) {
-    cb(err.toString())
-  }
+  return reshape({ parser }).process(html)
```

Comment:
it is passing on travis? the dep is set to `reshape/reshape` which pulls master if I'm not mistaken